src/Api/Response/ImageBannerResponse.php

```diff
@@ -23,19 +23,19 @@
 
     public function https()
     {
-        $sizes = $this->makeHttps($this->sizes);
+        $sizes = self::makeHttps($this->sizes);
 
         return new static(
             $this->alt, $sizes
         );
     }
 
-    private function makeHttps($urls)
+    private static function makeHttps($urls)
     {
         $sizes = [];
         foreach ($urls as $url) {
             foreach ($url as $k => $size) {
-                $sizes[$k] = 'https://www.wat.com/image/'.$k.'.jpg';
+                $sizes[$k] = str_replace(['http:/', 'internal_elife_dummy_api'], ['https:/', 'dummyapi.com'], $size);
```

Comment:
The API dummy does server HTTPS images locally, I think if we had some self signed https we could get around it. Alternatively I could move this behind debug flag